src/Strings.ts

```diff
@@ -27,6 +28,7 @@ const localizedStrings: LocalizedStrings = {
         messageFailed: "couldn't send",
         messageSending: "sending",
         timeSent: " at %1",
+        placeholder: "Type your message..."
```

Comment:
I did think about better names, but could not decide. Shall I make it consolePlaceholder? textboxPlaceholder?